src/main/scala/codacy/plugins/test/DockerHelpers.scala

```diff
@@ -3,10 +3,13 @@ package codacy.plugins.test
 import java.nio.file.{Files, Path}
 
 import codacy.TryExtension
-import codacy.docker.api._
 import codacy.plugins.docker.Pattern
+import com.codacy.analysis._
+import com.codacy.plugins.api.languages.{Language, Languages}
+import com.codacy.plugins.api.results.Tool
+import com.codacy.plugins.results.traits.DockerTool
+import com.codacy.plugins.utils.PluginHelper
 import play.api.libs.json.Json
-import plugins._
 
 import _root_.scala.sys.process._
 import scala.util.Try
```

Comment:
Most of the stuff here can be replaced by `DockerToolDocumentation` from codacy-plugins. I'll do that right away